### PR TITLE
[FEATURE] Register bitcoin addresses, begin polling

### PR DESCRIPTION
**Pull request by stevenzeiler**
[FOLD] Clean up, remove unnecessary dependency

### CODE REVIEW

File: models/token.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ var Token = bookshelf.Model.extend({
   balance: function () {
     return this.hasOne(Balance.model);
   },
+  addresses: function() {
+    return this.hasMany(Token.model);
```
Comment (wilsonianb) [inline models/token.js]: Why `hasMany` instead of `hasOne`?

Comment (stevenzeiler) [inline models/token.js]: In case there are multiple payment strategies supported by a host.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by codius-montecarlo** [comment]
The build is good and I'm seeing 1 +1s from the following folks:

* wilsonianb

As such, I'll attempt to merge and close this pull request.

**Pull request merged**